.github/workflows/tests-build-js.yml

```
@@ -23,7 +23,7 @@ jobs:
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
-        if: ${{ steps.check-artifact.outputs.exists != 'true' }}
+        if: ${{ steps.check-artifact.outputs.exists != 'true' && secrets.DOCKERHUB_USERNAME != '' }}
```

actionlint (1.7.11)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)